wikitemplate.md

```diff
@@ -189,8 +189,8 @@ _Each start should take less than 7 seconds_
 - [ ] Test that PDF is loaded over http at http://www.pdf995.com/samples/pdf.pdf
 - [ ] Test that https://mixed-script.badssl.com/ shows up as grey not red (no mixed content scripts are run).
 - [ ] Test that hovering the cursor over a link changes the cursor into a pointer (hand)
-- [ ] Visit https://dlptest.com/ftp-test/ and ensure that basic authentication is working by clicking on ftp://ftp.dlptest.com/ and ensuring you're prompted for credentials
 - [ ] Test that WebSockets are working by ensuring http://slither.io/ runs once "Play" has been clicked.
+- [ ] Visit https://www.httpwatch.com/httpgallery/authentication/#showExample10 and ensure "HTTP Basic Authenticaion" is working correctly when clicked on `Display Image` button. Follow the steps mentioned on page to verify it works correctly 
```

Comment:
small nits:

- "HTTP Basic Authenticaion" -> "HTTP Basic **Authentication**"
- is working correctly when **clicking** on *the* `Display Image` button.
- Follow the steps mentioned on **the** page to verify it works correctly.